tests/performance/locust_files/platform_ping_test.py

```diff
@@ -6,14 +6,18 @@
 import os
 from typing import NamedTuple
 
-import locust_plugins # necessary to use --check-fail-ratio 
+import locust_plugins
 from dotenv import load_dotenv
 from locust import task
 from locust.contrib.fasthttp import FastHttpUser
 
 logging.basicConfig(level=logging.INFO)
 
 
+# 'import locust_plugins' is necessary to use --check-fail-ratio
+assert locust_plugins  # nosec
+# adds assert here to avoid pycln removing it
+
```

Comment:
not sure I fully understand why adding the assert is requried

Reply:
improved doc